hawkular-inventory-api/src/test/java/org/hawkular/inventory/api/test/AbstractBaseInventoryPersistenceCheck.java

```diff
@@ -1629,52 +1676,52 @@ public void testBackendFind() throws Exception {
         CanonicalPath envPath = tenantPath.extend(Environment.class, "production").get();
         entity = backend.find(envPath);
         Environment env = backend.convert(entity, Environment.class);
-        Assert.assertEquals("com.acme.tenant", env.getTenantId());
+        Assert.assertEquals("com.acme.tenant", env.getPath().ids().getTenantId());
```

Comment:
I actually don't know if anything else broke API. Let's use revapi to find out :)